src/wallet/wallet2.cpp

```diff
@@ -13099,4 +13099,128 @@ uint64_t wallet2::get_bytes_received() const
+void wallet2::configure(const std::function<bool(const char*, std::vector<std::pair<const char*, boost::variant<uint64_t, uint32_t, std::pair<bool, uint64_t>, std::pair<uint32_t, std::vector<std::pair<uint32_t, std::string>>>, bool, std::string>>>&)> &f, const password_container &pwd_container)
+{
+  std::vector<std::pair<const char*, boost::variant<uint64_t, uint32_t, std::pair<bool, uint64_t>, std::pair<uint32_t, std::vector<std::pair<uint32_t, std::string>>>, bool, std::string>>> v;
+
+  v = {{tr("Always prompt before sending a transaction"), m_always_confirm_transfers}};
+  if (!f(NULL, v)) return;
+  m_always_confirm_transfers = boost::get<bool>(v[0].second);
+  rewrite(m_keys_file, pwd_container.password());
+
+  v = {{tr("Print ring members before sending a transaction"), m_print_ring_members}};
+  if (!f(NULL, v)) return;
+  m_print_ring_members = boost::get<bool>(v[0].second);
+  rewrite(m_keys_file, pwd_container.password());
+
+  v = {{tr("Store transaction secret keys in wallet cache for future use"), m_store_tx_info}};
+  if (!f(NULL, v)) return;
+  m_store_tx_info = boost::get<bool>(v[0].second);
+  rewrite(m_keys_file, pwd_container.password());
+
+  v = {{tr("Default priority"), std::make_pair(m_default_priority, std::vector<std::pair<uint32_t, std::string>>({{0, "Default"}, {1, "Unimportant"}, {2, "Normal"}, {3, "Elevated"}, {4, "Priority"}}))}};
+  if (!f(NULL, v)) return;
+  m_default_priority = boost::get<std::pair<uint32_t, std::vector<std::pair<uint32_t, std::string>>>>(v[0].second).first;
+  rewrite(m_keys_file, pwd_container.password());
+
+  v = {{tr("Automatically keep to date with the node"), m_auto_refresh}};
+  if (!f(NULL, v)) return;
+  m_auto_refresh = boost::get<bool>(v[0].second);
+  rewrite(m_keys_file, pwd_container.password());
+
+  v = {{tr("Refresh type"), std::make_pair(m_refresh_type, std::vector<std::pair<uint32_t, std::string>>({{RefreshFull, "Full"}, {RefreshOptimizeCoinbase, "Optimize coinbase"}, {RefreshNoCoinbase, "No coinbase"}}))}};
+  if (!f("Refresh type allows to you get faster updates if you know you don't mine to this wallet (choose 'No coinbase') or mine normally with a pool or solo (choose 'Optimize coinbase'). If unsure, select 'Optimize coinbase'.", v)) return;
+  m_refresh_type = (RefreshType)boost::get<std::pair<uint32_t, std::vector<std::pair<uint32_t, std::string>>>>(v[0].second).first;
+  rewrite(m_keys_file, pwd_container.password());
+
+  v = {{tr("Refresh from block height"), m_refresh_from_block_height}};
+  if (!f(NULL, v)) return;
+  m_refresh_from_block_height = boost::get<uint64_t>(v[0].second);
+  rewrite(m_keys_file, pwd_container.password());
+
+  v = {{tr("Confirm missing payment ID"), m_confirm_missing_payment_id}};
+  if (!f("If you sometimes have to send monero to a recipient which requires a payment ID, setting this option will prompt if you're sending without one in case you forgot to add one", v)) return;
```

Comment:
Could you rephrase this? I'm afraid it could be confusing (to users and translators)

Reply:
How about: setting this option will prompt when you send a payment without a payment ID. This can be useful if you sometimes need to use one and want to make sure you don't forget one when needed.

Reply:
This is not going in for 0.14.1 btw ^_^

Reply:
That would be much better IMO.
I guessed wasn't going in for 0.14.1 ;)